openedx/core/djangoapps/edx_global_analytics/utils/cache_utils.py

```diff
@@ -79,22 +78,23 @@ def get_cache_month_key():
     return '{0}-{1}-month'.format(year, month_number)
 
 
-def get_last_analytics_sent_date(type, token):
+def get_last_analytics_sent_date(query_type, token):
     """
     Get last analytics sent date from the cache.
     """
-    cache_key = 'ega_{}_{}'.format(type, token)
-    cache_value = cache.get(cache_key, datetime.fromtimestamp(0))
+    cache_key = 'ega_{}_{}'.format(query_type, token)
+    default_date = datetime.fromtimestamp(0)
```

Comment:
Do we need this separation on two lines?

Reply:
Done